spec/features/peep_spec.rb

```diff
@@ -0,0 +1,25 @@
+feature 'chitter welcome page' do
+  scenario 'view chitte welcome message' do
+    visit('/')
+    expect(page).to have_content('Chitter')
+  end
+end
+
+
+feature 'post' do
+  scenario 'post a peep to chitter' do
+    visit('/')
+    fill_in 'message', with: 'Hello!'
+    click_button('Submit')
+    expect(page).to have_content('Hello')
+  end
+end
+
+feature 'time' do
+  scenario 'see the time a peep was made' do
+    visit('/')
+    fill_in 'message', with: 'Hello!'
+    click_button('Submit')
+    expect(page).to have_content('Peep posted at:')
```

Comment:
It might be worth checking the time added is an instance of the Time class. Joe mentioned in peer check-in today that he'd done this, you can check it out [here](https://github.com/jmcnally17/chitter-challenge/blob/main/spec/peep_spec.rb).